frontend/src/pages/accounts/profile.test.tsx

```diff
@@ -835,7 +835,7 @@ describe("The dashboard", () => {
 
     const customAlias = screen.queryByText(/address3/);
     expect(customAlias).toBeInTheDocument();
-  });
+  }, 10000);
```

Comment:
And here too, not clear to me why this should be necessary?

Reply:
https://github.com/mozilla/fx-private-relay/pull/3748#discussion_r1294644891